lib/bpf/bpf.go

```diff
@@ -373,7 +373,15 @@ func (s *Service) emitCommandEvent(eventBytes []byte) {
 		}
 
 		argv := (*C.char)(unsafe.Pointer(&event.Argv))
-		buf = append(buf, C.GoString(argv))
+		argvStr := C.GoString(argv)
+		// Large arguments come in multiple events, those are tracked with a
+		// sequence number.
+		if event.Seq > 0 && len(buf) > 0 {
```

Comment:
Did you consider what happens when we overrun the ring buffer and lose some events? It looks like we may drop a segment of string in the middle this way, or even coalescing parts of different arguments. How damaging would this be?